cdc/owner/metrics.go

```diff
@@ -44,6 +44,13 @@ var (
 			Name:      "maintain_table_num",
 			Help:      "number of replicated tables maintained in owner",
 		}, []string{"changefeed", "capture", "type"})
+	changefeedStatusGauge = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "ticdc",
```

Comment:
seems the tag `ticdc`, `owner` should be const

Reply:
only consider each `changefeed`, owner is not specified in this case.

Reply:
sorry, i mean the tag in
```
Namespace: "ticdc",
Subsystem: "owner",
```